libraries/Microsoft.Bot.Builder.LanguageGeneration/LGTemplate.cs

```diff
@@ -64,20 +64,43 @@ public LGTemplate(LGFileParser.TemplateDefinitionContext parseTree, string sourc
         /// </value>
         public LGFileParser.TemplateDefinitionContext ParseTree { get; }
 
+        public override string ToString() => $"[{Name}]\"{Body}\"";
+
         private string ExtractBody(LGFileParser.TemplateDefinitionContext parseTree)
         {
-            var originText = parseTree.templateBody()?.GetText();
-            if (originText == null)
+            var result = string.Empty;
```

Comment:
I think we can pass LGResource into this, and extract body from orignal content, parseTree is not accurate